parking-lot-help-request/firmware/parking-lot-help-request-jsonata.jsonata

```diff
@@ -0,0 +1,19 @@
+(
+    $from := "+1800XXXXXXX";
+    $to := "+1404XXXXXXX";
+    $best_device_id := best_id ? best_id : device;
+    $body := function(){
+        (
+            $join([
+                "Alert! ",
+                $best_device_id,
+                " requires assistance. Help requested at ",
+                $fromMillis(when * 1000,
+                "[M01]/[D01]/[Y0001] [h#1]:[m01][P]",
+                "-0400"),
+                " ET."
+            ])
+        )
+    };
+    $result := "&Body=" & $body() & "&From=" & $from & "&To=" & $to & "&";
+)
```

Comment:
```suggestion
(
    $from := "+1800XXXXXXX";
    $to := "+1404XXXXXXX";
    $best_device_id := best_id ? best_id : device;
    $body :=
            $join([
                "Alert! ",
                $best_device_id,
                " requires assistance. Help requested at ",
                $fromMillis(when * 1000,
                "[M01]/[D01]/[Y0001] [h#1]:[m01][P]",
                "-0400"),
                " EDT."
            ]);
    $result := "&Body=" & $body & "&From=" & $from & "&To=" & $to & "&";
)
```
same comment as above.  I wish Markdown had some sort of `include/import` functionality so we didn't have to copy-pasta.

Reply:
updated, thanks